semconv/src/main/java/io/opentelemetry/semconv/resource/attributes/ResourceAttributes.java

```diff
@@ -666,6 +671,28 @@ public final class ResourceAttributes {
   public static final AttributeKey<String> WEBENGINE_DESCRIPTION =
       stringKey("webengine.description");
 
+  /** The name of the instrumentation scope - ({@code InstrumentationScope.Name} in OTLP). */
+  public static final AttributeKey<String> OTEL_SCOPE_NAME = stringKey("otel.scope.name");
```

Comment:
these are defined as resources https://github.com/open-telemetry/opentelemetry-specification/blob/a2a913a6dc72806749b2943ac64680cdc759dcde/semantic_conventions/scope/exporter/exporter.yaml#L2-L5, so are now generated in this file

Reply:
Can we leave the deprecated copies of these attributes in `SemanticAttributes` for the time being?